uplink/metainfo/client.go

```diff
@@ -408,6 +408,32 @@ func (client *Client) CommitObject(ctx context.Context, streamID storj.StreamID)
 	return Error.Wrap(err)
 }
 
+// GetObject gets single object
+func (client *Client) GetObject(ctx context.Context, bucket []byte, encryptedPath []byte, version int32) (_ storj.Object, _ storj.StreamID, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	response, err := client.client.GetObject(ctx, &pb.ObjectGetRequest{
+		Bucket:        bucket,
+		EncryptedPath: encryptedPath,
+		Version:       version,
+	})
+	if err != nil {
+		return storj.Object{}, storj.StreamID{}, Error.Wrap(err)
+	}
+
+	object := storj.Object{
+		Bucket: storj.Bucket{
+			Name: string(response.Object.Bucket),
+		},
+		Path:    string(response.Object.EncryptedPath),
+		Created: response.Object.CreatedAt,
+		Expires: response.Object.ExpiresAt,
+		// TODO custom type for response object or modify storj.Object
```

Comment:
How about size, metadata and other fields?

Reply:
Currently, we are not storing `Object` anywhere so only some information can be extracted from segment. `storj.Object` doesn't fit well into new Metainfo structs e.g. metainfo is `map[string]string` instead of `[]byte`. I believe before we will finish whole Metainfo refactoring we still need to adjust client-side objects. That's why I left TODO.

Reply:
OK. I see.